src/Microsoft.Health.Fhir.Api.UnitTests/Controllers/ExportControllerTests.cs

```diff
@@ -73,7 +73,9 @@ private ExportController GetController(ExportConfiguration exportConfig)
             optionsOperationConfiguration.Value.Returns(operationConfig);
 
             return new ExportController(
-                Substitute.For<IFhirRequestContextAccessor>(),
+                mediator ?? Substitute.For<IMediator>(),
```

Comment:
Since the ExportController makes sure these are not null, having null as default value doesn't make sense. I would extract these into member variable and just initialized them as mock in the constructor. If the test needs to mock certain behavior, it can does that or otherwise leave the default as is.